src/test/groovy/ca.cutterslade.gradle.analyze/AnalyzeDependenciesPluginSpec.groovy

```diff
@@ -226,7 +251,7 @@ class AnalyzeDependenciesPluginSpec extends Specification {
         GradleRunner.create()
                 .withProjectDir(projectDir.getRoot())
                 .withPluginClasspath()
-                .withArguments("build")
+                .withArguments("-i", "--stacktrace", "--no-parallel", "build")
```

Comment:
Should these new options be conditional?

Reply:
I'm not sure, it was impossible to figure out test failures without the info level logging.